code/modules/cargo/exports/large_objects.dm

```diff
@@ -138,4 +138,5 @@
 	worth += gases[/datum/gas/miasma][MOLES]*10
 	worth += gases[/datum/gas/tritium][MOLES]*5
 	worth += gases[/datum/gas/pluoxium][MOLES]*5
+	worth += gases[/datum/gas/hydrogen][MOLES]*10
```

Comment:
I'm no cargo main but it might be worth considering making this a bit higher, at least if you don't add it to lavaland.

Reply:
done next commit, increased to 25